### ISSUE TITLE
Членские билеты

### ISSUE THREAD

**Issue by NightEagle14** [alxdrlitreev/MovementCharterProject#4]
> 8.5. Членам Движения выдаются членские билеты установленного Федеральным Советом Движения образца.

Предлагаю передать полномочия по установке образца членского билета в Региональные Советы, не придерживаться единого образца. Плюс - не нужно будет пересылать их по всей стране, а каждый напечатает свои.

**Comment by alxdrlitreev** [comment]
> Предлагаю передать полномочия по установке образца членского билета в Региональные Советы, не придерживаться единого образца. Плюс - не нужно будет пересылать их по всей стране, а каждый напечатает свои.

Соглашусь, но какие-то общие требования к ним нужно предъявлять.

**Comment by NightEagle14** [comment]
> Соглашусь, но какие-то общие требования к ним нужно предъявлять.

Окей.

На членском билете должны быть изображены название движения, федеральная и/или региональная символика и уникальный номер членского билета.

**Comment by alxdrlitreev** [comment]
> На членском билете должны быть изображены название движения, федеральная и/или региональная символика и уникальный номер членского билета.

Согласен, внесём это в устав. Надо еще про вносимые туда данные уточнить будет.
